assets/src/js/header.js

```diff
@@ -149,4 +151,13 @@ export const setupHeader = () => {
   if (closeNavMenuButton) {
     closeNavMenuButton.onclick = event => closeElement(event, '.nav-menu-toggle');
   }
+
+  let searchFocused = false;
```

Comment:
This is to send the event not more than once. We only use it to filter out sessions with at least one interaction with the search box.